swabbie-web/src/main/kotlin/com/netflix/spinnaker/swabbie/controllers/ResourceController.kt

```diff
@@ -34,8 +32,26 @@ class ResourceController(
   private val applicationEventPublisher: ApplicationEventPublisher,
   private val workConfigurations: List<WorkConfiguration>
   ) {
+
   @RequestMapping(value = ["/marked"], method = [RequestMethod.GET])
-  fun markedResources(): List<MarkedResource> = resourceTrackingRepository.getMarkedResources()
+  fun markedResources(
+    @RequestParam(required = false) expand: Boolean?
```

Comment:
I think you could be able to do something like this instead, which will eliminate the null check...

`@RequestParam(required = false, default = "false") expand: Boolean`